scipy/stats/tests/test_distributions.py

```diff
@@ -4946,6 +4954,45 @@ def test_distribution_outside_support(
             np.full(len(points), expected)
         )
 
+    @pytest.mark.parametrize(
+        'alpha,beta,expected',
+        [
+            (1.77207328046188083981, 0.50593731369029959488, 0.27893263679826763024),
+            (1.92170015224102352924, -0.87794427466859259468, 0.28105475720231620773),
+            (1.56548060516336340342, -0.40162203419113917668, 0.27128213319420380056),
+            (1.74208034477843876076, -0.38180029468259246705, 0.28020219924424716318),
+            (1.57480025276899127107, -0.25200194914153684067, 0.28013657621866516756),
+        ]
+    )
+    def test_x_near_zeta(
+            self, alpha, beta, expected
+    ):
+        """Ensure the pdf/cdf routines do not return bad values when x near zeta.
```

Comment:
Actually, I generated the expected values with the fixed version of scipy code. Could I assume that it was really fixed correctly and use this values? Should I double-check the values using Nolan's C program or some else?

Reply:
Yes, out of principle you should compare to something external.

Reply:
OK. I don't want to use Nolan's program, because a free version of his program is Windows-only and closed-source. So I tested in an online version of [MATLAB](https://www.mathworks.com/help/stats/stable-distribution.html) (ironically, isn't?) using such code:
```matlab
format long

alpha = 1.5748002527689913;
beta = -0.25200194914153684;
expected = 0.28013657621866517;

pd = makedist('Stable','alpha',alpha,'beta',beta,'gam',1,'delta',0);
% we need to adjust x. It is the same as x = 0 In scipy.
x = -beta * tan(pi * alpha / 2)
pd.pdf(x)
assert (expected - pd.pdf(x) <= eps) % eps is constant - machine epsilon
```

All tests have passed successfully. Is it good enough? Do I still need to add the comment?

Reply:
MATLAB’s as a reference implementation is fine. I think you should change the expected values in the relevant SciPy test to the values produced by MATLAB, and leave a comment stating where the values came from.